ledger/sandbox/src/main/scala/db/migration/postgres/V26_1__Fill_create_argument.scala

```diff
@@ -5,7 +5,7 @@ package db.migration.postgres
 
 import java.io.ByteArrayInputStream
 
-import com.daml.platform.store.serialization.{ContractSerializer, ValueSerializer}
+import com.daml.platform.store.serialization.ValueSerializer
```

Comment:
How come you no longer have to import `ContractSerializer`?

Reply:
I see from the builds that you _do_. 🙃

Reply:
Good question, looks like the compiler got there before you did. :slightly_smiling_face:

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5963/commits/3a1e438ec0990875ccdd06bd83e40728340ea4d4